### PR TITLE
1370 iframe sizing

### PR DESCRIPTION
**Pull request by nmorduch**
I just merged the new commit to staging, so it can be tested there. I think this is messy in the blog posts but I want to check in with @mariaelkin about whether that should be a blocker to deployment.

Not sure why tests aren't passing (maybe because I merged to staging without waiting for them? probably not?)

PR to master for the purpose of the diff, but I want to start deploying production by merging `staging` into `master` so that staging is actually staging. (Yes I still want a dev server one day so that staging isn't filling that role, but for now we can just stage one change at a time.)

### CODE REVIEW

**Comment by nmorduch** [comment]
Some places I've been testing this on staging:

Report (working perfectly)
- https://staging.newamerica.org/education-policy/reports/undermining-pell-iv/introduction/
- https://staging.newamerica.org/education-policy/reports/undermining-pell-iv/print/

Blog post and about page (only works for column-width; i think the correct solution at least for now is just to make a note about this in the admin)
- https://staging.newamerica.org/public-interest-technology/blog/ai-marketing-trojan-horse/
https://staging.newamerica.org/bretton-woods-ii/about/

**Comment by nmorduch** [comment]
Small changes for rereview @kevinhowbrook

File: newamericadotorg/blocks.py [hunk 1]
```diff
@@ -69,7 +69,7 @@ class IframeBlock(blocks.StructBlock):
 		('column-width', 'Column Width (max 650px)'),
 		('width-1200', 'Site Width (max 1200px)'),
 		('full-width', 'Full Width (max 100%)')
-	], default='column-width', required=False, help_text='The maximium width of the iframe. For pages made in 2018 and earlier,  "Column Width" will maintain the original design.')
+	], default='column-width', required=False, help_text='The maximum width of the iframe. Always use "Column Width" for non-report content types (e.g. blog posts, About pages). Never use "Full-Width" unless specifically instructed to by your designer.')
```
Comment (kevinhowbrook) [inline newamericadotorg/blocks.py]: I think you will need an extra migration for this,  or regenerate the last one with this new value

**Comment by kevinhowbrook** [comment]
> Small changes for rereview @kevinhowbrook

Updates look good, could you just double check you aren't missing migrations though after [updating the help text](https://github.com/newamericafoundation/newamerica-cms/pull/1380/commits/db4f84921351565ce66cb6bfeb8c665ca1972adc#diff-2584ab4d74c724b82b6e31b03243f6b9). As it looks like the [migration file](https://github.com/newamericafoundation/newamerica-cms/pull/1380/files#diff-84f02945122b37eb1a44ed39412cff84) has different help text

**Comment by nmorduch** [comment]
Closing in favor of #1384

**Pull request closed without merging**